arctic/_compression.py

```diff
@@ -1,13 +1,17 @@
 import os
 import logging
-from . import _compress as clz4
+from multiprocessing.pool import ThreadPool
 
+from lz4.block import compress as lz4_compress, decompress as lz4_decompress
 
 logger = logging.getLogger(__name__)
 
 # switch to parallel LZ4 compress (and potentially other parallel stuff), Default True
 ENABLE_PARALLEL = not os.environ.get('DISABLE_PARALLEL')
-LZ4_N_PARALLEL = 50  # No. of elements to use parellel compression in LZ4 mode
+LZ4_N_PARALLEL = os.environ.get('LZ4_N_PARALLEL', 50)  # No. of elements to use parallel compression in LZ4 mode
+LZ4_WORKERS = os.environ.get('COMPRESSION_WORKERS', 4)
```

Comment:
I thought `2` workers was a reasonable sweet spot for this?

Reply:
it is 2x workers for non-HC, and 4 (or better 8) for HC.

We could either have 2 thread pools (one for default with 2x and one for HC with 8x) which I don't like.
So I chose a middle ground of 4.

Since VersionStore doesn't use HC, the users can specify env:  COMPRESSION_WORKERS=2
For TickStore heavy use, users may set COMPRESSION_WORKERS=8.

I might add this as a comment somewhere